### DISCUSSION TITLE
Collapsable Navigation

### DISCUSSION THREAD

**Discussion by justified1gmail** [Videogamers0/StoryManager#5] [Ideas]
It would be nice to be able to hide or collapse the story navigation area on the left side.  
It is sizeable, but can't shrink enough to hide it.

When reading a story, seeing it is a distraction.

**Comment by Videogamers0** [comment]
f9c6174a7a8ee99b4bb3b24fb5d392f567176ffa

Sure, version 1.0.3 now has an expander on the sidebar. 
https://github.com/Videogamers0/StoryManager/releases/latest

**Comment by justified1gmail** [comment]
That's fantastic.  Thank you
